packages/components/src/Disclosure/Disclosure.mdx

```diff
@@ -0,0 +1,114 @@
+---
+name: Disclosure
+menu: Components
+route: /components/disclosure
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { Content } from "@jobber/components/Content";
+import { Disclosure } from ".";
+
+# Disclosure
+
+```ts
+import { Disclosure } from "@jobber/components/Disclosure";
+```
+
+<Playground>
+  <Disclosure title="Open to see more content organa palpatine lobot jawa kamino.">
+    <Content>
+      <p>
+        Lucas ipsum dolor sit amet darth sebulba solo vader yoda bothan wedge
+        kenobi dantooine hutt.
+      </p>
+      <p>
+        Moff qui-gonn darth leia solo hutt gonk darth. Skywalker tatooine
+        dagobah thrawn.
+      </p>
+    </Content>
+  </Disclosure>
+</Playground>
+
+## Props
+
+<Props of={Disclosure} />
+
+## Usage Guidelines
+
+The `<Disclosure>` will reveal and hide additional information that our users
+may care about, should they choose to learn more about it.
+
+## Content Guidelines
+
+- **Title** should be informative and label the type of content grouped in the
+  body content in it
+  - uses `<Heading level={4}>` component
+- **Content** should be actionable and clear. May contain:
+  - good ol’ plain text
+  - accepts any React component as a child
+  - except: `<Page>`, `<Table>`
+
+## Accessibility
+
+- Users should be able to use their keyboard and toggle the component's
+  open/close state
+- Headings are permitted in `<summary>` elements to provide in-page navigational
+  assistance
+- Include aria-expanded attribute on the trigger to communicate to assistive
+  technology
+- Include aria-controls attribute ties on the trigger to the content it controls
+  using the id of the collapsible container
+- The trigger contains a downward-pointing-arrow to hint that it can be
+  expanded. when the disclosure item is in expanded state, this is rotated 180
+  degrees to point upwards
+- The icon will be given an `aria-hidden="true"` attribute to hide it from
+  assistive technologies, as well as `focusable="false"` to address an
+  inconsistency in IE and older versions of Edge
+
+## Responsiveness
+
+- This component should handle both click and touch, as well as keyboard inputs
+- This component should fill the width of its container
+- If the component is less than `375px` wide, the title will wrap and should not
+  be truncated
+- Text increases/decreases when user zooms in/out
+
+## Mockup
```

Comment:
We can remove the entire mockup section now that we have a working-in-code version 😃